### ISSUE TITLE
upgrade mediation to assign n channels

### ISSUE THREAD

**Issue by combatopera** [combatopera/pym2149#41]
this is needed for dosound, e.g. if there are 4 channels we'd like it to be possible to have a dosound note on the first 3 and a regular Note on the 4th. this depends on #38 so that a Note can declare which channels it wants even in the Unpitched case

**Comment by combatopera** [comment]
for lc lang it will be natural for a dosound instrument to use the assigned channel and the next two (with wrap-around). perhaps revisit this ticket after #54

**Comment by combatopera** [comment]
possibly wontfix due to #73 - indeed
